tests/http/static-files.test.js

```diff
@@ -11,233 +11,249 @@ var types = {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/images/internet-graphic.e9a5980f4251c71bdd72d088f80d9864.svg': {
-      type: types.svg,
-      cache: cache.long
-    },
-    '/assets/images/mozilla-circular.911f4f7f4e6682c9893b8441d2e09df40cea80e2.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/images/mozilla.1068965acefde994a71c187d253aca2b.svg': {
-      type: types.svg,
-      cache: cache.long
-    },
-    '/assets/images/payment_logos_updated.4ca531c0b90d3635aef0c1fc78d5967e.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/images/payment_logos_discover.e08ebedbd623af15a14094b6821ca137.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/images/paypal_logo@2x.603485f928e450e7098eb55bc982e19a.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/js/jquery.min.js': {
-      type: types.js,
-      cache: cache.long
-    },
-    '/assets/js/placeholder.min.js': {
-      type: types.js,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy01.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy02.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy03.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy04.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy05.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy06.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy07.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy08.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy09.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/assets/tiles/eoy10.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/en-US/': {
-      type: types.html,
-      cache: cache.short
-    },
-    '/en-US/about/': {
-      type: types.html,
-      cache: cache.short
-    },
-    '/en-US/give-bitcoin/': {
-      type: types.html,
-      cache: cache.short
-    },
-    '/en-US/share/': {
-      type: types.html,
-      cache: cache.short
-    },
-    '/en-US/thank-you/': {
-      type: types.html,
-      cache: cache.short
-    },
-    '/favicon.png': {
-      type: types.png,
-      cache: cache.long
-    },
-    '/main.asdf.js': {
-      type: types.js,
-      cache: cache.long
-    },
-    '/style.asdf.css': {
-      type: types.css,
-      cache: cache.long
-    }
-  };
+  before(async(done) => {
+    instance = await server();
+    done();
+  });
+
+  Object.keys(files).forEach((url) => {
+    it(`should return okay when accessing ${url}`, async(done) => {
+      let response = await instance.inject({ url });
+      let { type, cache, vary } = files[url];
 
-  var instance = server({ useDomains: false });
```

Comment:
useDomains: false is a thing we don't need anymore?

Reply:
Deprecated feature